rendermodules/em_montage_qc/plots.py

```diff
@@ -136,10 +138,14 @@ def plot_defects(render, stack, out_html_dir, args):
         #else:
         #    residual.append(50) # a high value for residual for that tile
 
-    out_html = tempfile.NamedTemporaryFile(suffix=".html", delete=False, mode='w', dir=out_html_dir)
-    out_html.close()
+    out_html = os.path.join(
+            out_html_dir,
+            "%s_%d_%s.html" % (
+                stack,
+                z,
+                datetime.datetime.now().strftime('%Y%m%d%H%S%M%f')))
 
-    output_file(out_html.name)
+    output_file(out_html)
```

Comment:
I am just wondering if this works... the integration tests doesn't check if the returned value is a html file name and not a tempfile object.

Reply:
it isn't a file object, it is the path:
https://github.com/AllenInstitute/render-modules/blob/c860bf7b4719c576b7804a7a6337d76f28052963/rendermodules/em_montage_qc/plots.py#L141-L146